ui/imports/utils/Constants.qml

```diff
@@ -238,6 +238,12 @@ QtObject {
         readonly property int moreThanFiveMins: 4
     }
 
+    readonly property QtObject translationsState: QtObject {
```

Comment:
this could easily be a QML `enum`

Reply:
Yes, it could :D Just take a look at the wider context of `Constants.qml` -> "when in Rome, do as the Romans do"

Reply:
Yeah and then Rome collapsed :D